### PR TITLE
feat: kotakt footer

### PR DESCRIPTION
**Pull request by Bateyjosue**
### Description
In this PR, We work on: 
- [ ] Title with
- [ ] contact Button
- [ ] footer with logo with
- [ ] footer navigation
### Screenshot

kotakt1

## Cheers 🎉 🎉 🎉

### CODE REVIEW

**Review by HIRWA13 — CHANGES_REQUESTED** [review]
measurements are good but try to match the same design as it is designed on the figma!

footer review

**Comment by Bateyjosue** [comment]
Hey @HIRWA13, I have seen your comments on this PR.  but I'm A bit confused because i don't know which measurement you are talking about. 
Please be Specific

**Thanks For talking time to review this PR**

##

> Don't forget to tag me on @Bateyjosue so that I get notified

**Review by HIRWA13 — APPROVED** [review]
Hi @Bateyjosue, thanks for applying the changes that were missing on the footer.!
After reviewing both the footer and the kontakt section I can see that nothing is missing there.

you can merge this branch to dev.

File: src/Components/kotakt.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="bg-white transform skew-y-3 ">
```
Comment (Kevinemug) [inline src/Components/kotakt.html:1]: I liked how you perfected the skewing dimensions

**Review by Kevinemug — APPROVED** [review]
hello @Bateyjosue  I approved your PR

**Pull request merged**